src/components/header/header.jsx

```diff
@@ -18,7 +19,14 @@ const Header = () => {
 					<h1>Need Games</h1>
 				</Link>
 			</div>
-			<input type='search' className='text-field' placeholder='Search For Games' />
+			<input
+				type='search'
+				className='text-field'
+				placeholder='Search For Games'
+				onChange={(e) => {
```

Comment:
I think it's better to use onSubmit here as the filterDispatch gets called on every keystroke for onChange.

Reply:
I'll do this in next submission as I have very less products to filter it out from.